### PR TITLE
docs(ingest): add example of ssl with mysql

### PR DESCRIPTION
**Pull request by hsheth2**
## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/docs/sources/mysql/mysql_recipe.yml [hunk 1]
```diff
@@ -9,5 +9,12 @@ source:
     username: root
     password: example
 
+    # If you need to use SSL with MySQL:
```
Comment (jinlintt) [inline metadata-ingestion/docs/sources/mysql/mysql_recipe.yml:12]: I think this is more general than just MySQL.
Maybe we can create a dedicated page that explains how to configure SSL connection parameters?

Comment (hsheth2) [reply metadata-ingestion/docs/sources/mysql/mysql_recipe.yml:12]: Unfortunately the `connect_args` piece is db driver specific, so it doesn't generalize very well - we'll have to go through the sources one by one to add SSL examples

Comment (jinlintt) [reply metadata-ingestion/docs/sources/mysql/mysql_recipe.yml:12]: Got it. Thank you!

**Review by jinlintt — APPROVED** [review]

**Review by anshbansal — APPROVED** [review]
LGTM

**Pull request merged**